src/internal/views/nunjucks/billing/batch-invoice.njk

```diff
@@ -72,18 +72,90 @@ invoiceDebitsTotal %}
 
       {{ batchMetadata(batch, invoice)}}
 
+      <!-- Bills linked to reissue -->
+      {% macro displayBillingState(inv, billingBatchId) %}
+        <!-- Is "rebill" (as opposed to "rebilled") in invoices -->
+        {% if inv.rebillingStateLabel == 'rebill' and inv.originalInvoiceId != null %}
+          Reissued <span class="screen-reader-only">bill</span>
+        {% endif %}
+
+        {% if inv.rebillingStateLabel == 'rebilled' and inv.originalInvoiceId != inv.id %}
+          Reissue <span class="screen-reader-only">bill</span>
+        {% endif %}
```

Comment:
This could be changed to 
`{% if inv.rebillingStateLabel == 'rebill' or inv.rebillingStateLabel == 'rebilled' %}
          Reissued <span class="screen-reader-only">bill</span>
  {% endif %}`